stories/Molecules/Blocks/CountryCard/countrycard.scss

```diff
@@ -14,10 +14,16 @@
 }
 
 .country-card {
-  overflow: hidden;
+  &.glide__slide{
+    height: fit-content;
```

Comment:
@Ashish-srij Any specific reason we need `height: fit-content`? This has a very minimal support in Firefox (https://caniuse.com/?search=fit-content). Kindly test it on Firefox and fix this if required

Reply:
@shishirphadke Fix with different solution.

Reply:
Ok. Looks good! Thanks.